tests/Composer/ComposerLockTest.php

```diff
@@ -119,24 +119,24 @@ public function it_errors_when_package_is_not_found()
     public function it_can_gets_normalized_version_of_a_package_from_composer_lock()
     {
         PackToTheFuture::generateComposerLockForMultiple([
-            'package/one' => '1.0.0', // Doesn't need normalizing.
+            'package/one' => '1.0.0',
             'package/two' => '2.0',
             'package/three' => '3',
             'package/four' => 'v4.0.0',
             'package/five' => 'v5.0',
             'package/six' => 'v6',
-            'package/seven' => '7.1.0-beta.1', // Doesn't need normalizing.
+            'package/seven' => '7.1.0-beta.1',
             'package/eight' => 'v8.1.0-beta.1',
         ], $this->lockPath);
 
-        $this->assertEquals('1.0.0', Lock::file()->getInstalledVersion('package/one'));
-        $this->assertEquals('2.0.0', Lock::file()->getInstalledVersion('package/two'));
-        $this->assertEquals('3.0.0', Lock::file()->getInstalledVersion('package/three'));
-        $this->assertEquals('4.0.0', Lock::file()->getInstalledVersion('package/four'));
-        $this->assertEquals('5.0.0', Lock::file()->getInstalledVersion('package/five'));
-        $this->assertEquals('6.0.0', Lock::file()->getInstalledVersion('package/six'));
-        $this->assertEquals('7.1.0-beta.1', Lock::file()->getInstalledVersion('package/seven'));
-        $this->assertEquals('8.1.0-beta.1', Lock::file()->getInstalledVersion('package/eight'));
+        $this->assertEquals('1.0.0.0', Lock::file()->getNormalizedInstalledVersion('package/one'));
+        $this->assertEquals('2.0.0.0', Lock::file()->getNormalizedInstalledVersion('package/two'));
+        $this->assertEquals('3.0.0.0', Lock::file()->getNormalizedInstalledVersion('package/three'));
+        $this->assertEquals('4.0.0.0', Lock::file()->getNormalizedInstalledVersion('package/four'));
+        $this->assertEquals('5.0.0.0', Lock::file()->getNormalizedInstalledVersion('package/five'));
+        $this->assertEquals('6.0.0.0', Lock::file()->getNormalizedInstalledVersion('package/six'));
+        $this->assertEquals('7.1.0.0-beta1', Lock::file()->getNormalizedInstalledVersion('package/seven'));
```

Comment:
Shouldn't this still be `beta.1`?

Reply:
Nope composer normalizes it to without the dot.